core/arch/arm/kernel/elf_load.c

```diff
@@ -38,6 +38,9 @@
 #include "elf32.h"
 #include "elf64.h"
 
+/* OP-TEE does not have SIZET_MAX */
+#define SIZET_MAX ((size_t)~((size_t)0))
```

Comment:
The name of the define should be `SIZE_MAX` and should be located in `stdint.h` according to C99. The macro should also be in a form that it can be used in `#if` expressions.